code/modules/mining/equipment/lazarus_injector.dm

```diff
@@ -26,18 +26,21 @@
 				to_chat(user, "<span class='info'>[src] does not work on this sort of creature.</span>")
 				return
 			if(M.stat == DEAD)
-				M.faction = list("neutral")
 				M.revive(full_heal = 1, admin_revive = 1)
-				if(ishostile(target))
-					var/mob/living/simple_animal/hostile/H = M
-					if(malfunctioning)
-						H.faction |= list("lazarus", "[REF(user)]")
-						H.robust_searching = 1
-						H.friends += user
-						H.attack_same = 1
-						log_game("[key_name(user)] has revived hostile mob [key_name(target)] with a malfunctioning lazarus injector")
-					else
-						H.attack_same = 0
+				if(M.mind)
+					M.AIStatus = AI_OFF // don't let them attack people randomly after revived
+				else // only do this to mindless mobs
+					M.faction = list("neutral")
```

Comment:
Why do we only want to set the faction for mindless mobs?

Reply:
yeah I think it's better to remove

Reply:
I think this is intended behaviour, to make the simple mob non agressive, mindless doesn't mean it has no AI